scripts/tool_shed/api/export.py

```diff
@@ -101,14 +101,11 @@ def main(options):
             src = None
             dst = None
             try:
-                src = urllib2.urlopen(download_url)
-                dst = open(file_path, 'wb')
-                while True:
-                    chunk = src.read(CHUNK_SIZE)
-                    if chunk:
-                        dst.write(chunk)
-                    else:
-                        break
+                src = requests.get(download_url, stream=True)
+                with open(file_path, 'wb') as handle:
```

Comment:
You need to remove the lines 114-115:
```python
                 if dst:
                     dst.close()
```

Also I'd keep the variable name `dst` (not really important any way).